### PR TITLE
User Locations Fix

### PR DESCRIPTION
**Pull request by EvanBarbour3**
## Bug Fixes
- Fixed User Locations for super users
- Added a previously removed use statement for Collection

[Delivers#110698174]

### CODE REVIEW

**Comment by wnowicki** [comment]
You are mixing some concerns there. Please move stuff related to *Locations* to separate methods, don't mix them with standard *User* edit and create actions.

**Comment by EvanBarbour3** [comment]
@wnowicki Took a standard index approach like in Controller. Thoughts?

File: app/Http/Controllers/Controller.php [hunk 1]
```diff
@@ -148,18 +150,61 @@ protected function fetchApplicationDetails($id)
...
+     * @return Collection
+     * @throws RedirectException
+     */
+    protected function fetchMerchantLocations($user)
+    {
+        try {
+            $merchant = Merchant::findOrFail($user->merchant_id);
+            $installations = $merchant->installations()->get();
+        } catch (ModelNotFoundException $e) {
+            if($user->id == 1) {
```
Comment (wnowicki) [inline app/Http/Controllers/Controller.php]: What do you mean by user `1`? What if we have more `su` users or even our default one would be with `id = 2` ??

Let me know once you'll finish your lunch.

**Comment by wnowicki** [comment]
is it really ready ?

**Comment by EvanBarbour3** [comment]
@wnowicki Yeah, really, ready! 😂

**Comment by wnowicki** [comment]
Happy.
Please do marge for me. As I'm tryng to test `framework` change on TEST.

**Pull request merged**